### ISSUE TITLE
[C#6] Support `using static`

### ISSUE THREAD

**Issue by DanTup** [bridgedotnet/Bridge#322]
I'm not sure which version of C# you're currently tracking, but `using static` would be really useful for creating bindings for JS that has global functions (for example, Jasmine). Currently if you try to use this, you get a compile error.

This code works fine:

```
using DanTup.Bridge.Jasmine;

namespace SampleTests.Tests
{
	public class SimpleTests
	{
		[Ready]
		public static void Test1()
		{
			Jasmine.Describe("A suite", () =>
			{
				Jasmine.It("contains spec with an expectation", () =>
				{
					Jasmine.Expect(true).ToBe(true);
				});
			});
		}
	}
}
```

However if you try importing the `Jasmine` class statically:

```
using static DanTup.Bridge.Jasmine.Jasmine;

namespace SampleTests.Tests
{
	public class SimpleTests
	{
		[Ready]
		public static void Test1()
		{
			Describe("A suite", () =>
			{
				It("contains spec with an expectation", () =>
				{
					Expect(true).ToBe(true);
				});
			});
		}
	}
}
```

You get the following build error:

```
The "GenerateScript" task failed unexpectedly.
System.NullReferenceException: Object reference not set to an instance of an object.
   at Bridge.Contract.EmitterException.get_FileName()
   at Bridge.Build.GenerateScript.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__1.MoveNext()	SampleTests		
```

```
All packages are already installed and there is nothing to restore.
1>------ Build started: Project: SampleTests, Configuration: Debug Any CPU ------
1>  SampleTests -> C:\Work\Source\DMS\DanTup.Bridge.Jasmine\SampleTests\bin\SampleTests.dll
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018: The "GenerateScript" task failed unexpectedly.
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object.
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018:    at Bridge.Contract.EmitterException.get_FileName()
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018:    at Bridge.Build.GenerateScript.Execute()
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
1>C:\Work\Source\DMS\DanTup.Bridge.Jasmine\packages\Bridge.1.7.0\build\Bridge.targets(12,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__1.MoveNext()
========== Build: 0 succeeded, 1 failed, 1 up-to-date, 0 skipped ==========
```

**Comment by geoffreymcgill** [comment]
The `using static` syntax functionality is part of the C# 6 language spec and will be included in Bridge 2.0.

**Comment by LeonidVeriga** [comment]
Related #1435
